### PR TITLE
Added Github Action workflow script

### PR DESCRIPTION
**Pull request by dulovics**
Replacing the Travis CI pipline with Github Actions.

## Description
Currently the Travis CI pipeline build isn't working due to the pricing schema change and needs to replace with Github Actions.
As part of this PR I also documented the python 3.9 support.

## Checklist

- [X] Changes are covered by tests.
- [X] Documentation has been updated with the new changes.
- [X] Readme is still relevant after the changes has been introduced.
- [X] Library version is updated in `setup.py`.  
      _Required to make a new release._

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            python: 3.6
+            upload: true
+    steps:
+      # Checks-out the repository under $GITHUB_WORKSPACE
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Cache pip check
```
Comment (vikahl) [inline .github/workflows/main.yml:34]: Just curious, what does this step do?

Comment (dulovics) [reply .github/workflows/main.yml:34]: This action sets up a Python environment for use by the actions, more details can be found in [here](https://github.com/actions/setup-python).
In our case is responsible for setting up a workflow for each python version what we are supported.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8, 3.9]
+        include: # define single matrix case that performs the upload
+          - os: ubuntu-latess
```
Comment (mspringare) [inline .github/workflows/main.yml:24]: is this a type or is it called latess?

Comment (dulovics) [reply .github/workflows/main.yml:24]: It was a typo, I will fix it 👍

Comment (mspringare) [reply .github/workflows/main.yml:24]: cool, same in comet-core 👍🏼

**Review by mspringare — APPROVED** [review]
I am all for trying this. Our current CI setup doesnt work any way. 👍🏼

**Pull request merged**